src/components/ThemeRegistry/colors.js

```diff
@@ -100,6 +100,6 @@ const colorsDefault = Object.fromEntries(
 
 export const colors = {
   ...colorsDefault,
-  grey: { ...grey, light: grey[500], main: grey[600], dark: "#000000", contrastText: "#FFFFFF" },
+  grey: { ...grey, light: grey[400], main: grey[600], dark: "#000000", contrastText: "#FFFFFF" },
```

Comment:
Hm, I would keep it as it was, as this setting has side effects, grey button hover color, for example.